.rubocop.yml

```diff
@@ -7,6 +7,7 @@ AllCops:
   - "vendor/**/*"
   - "db/**/*"
   - "client/**/*"
+  - "app/jobs/simulate_bus_location_job.rb"
```

Comment:
Oh, yeah, I had meant to rework things so the complexity metrics wouldn't actually break the build on Circle, since we do have CodeClimate. In the meantime if a metrics cop is giving you trouble you can [disable it with a magic comment](http://rubocop.readthedocs.io/en/latest/configuration/#disabling-cops-within-source-code) (here's the [list of cop names](http://rubocop.readthedocs.io/en/latest/cops_metrics/)). That way you still get the benefit of the style checking.